.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ env:
 jobs:
 
   check-changelog:
-    if: github.ref != 'refs/heads/main'
+    if: false && github.ref != 'refs/heads/main'
```

Comment:
Was this for debugging, or is it necessary for the fix to the release process?

Reply:
Yeah the checker itself is no longer in sync with the output. And the checker doesn't seem to expose options to change that - https://www.npmjs.com/package/cae-release-notes-format-checker.

This at least allows us to keep the infrastructure so that if @JakeChampion updates the checker we can then reenable it.